README.md

```diff
@@ -31,16 +31,20 @@ Or install it yourself as:
 
     $ gem install npmdc
 
+To use npmdc with yarn you should install yarn.
+
 ## Usage
 
 ### Rails
 
 ```ruby
 YourApp::Application.configure do
+  config.npmdc.package_manager  = "yarn"                            # `npm` by default
+  config.npmdc.path_to_yarn     = "/path/to/your/yarn"              # nil by default, so the command is just `yarn`
   config.npmdc.path             = "/path/to/your/frontend/code/dir" # `Rails.root` by default
   config.npmdc.format           = "doc"                             # `short`, `doc`, `progress`. `short` by default
   config.npmdc.color            = false                             # `true` by default
-  config.npmdc.types            = ["dependencies"]                  # `["dependencies", "devDependencies"]` by default
+  config.npmdc.types            = ["dependencies"]                  # `["dependencies", "devDependencies"]` by default (ignored with yarn package_manager)
```

Comment:
which one is ignored by yarn? Can we add this information explicitly to a separate README section about pros and cons of the both tools? /cc @lysyi3m